pkg/volume/rbd/rbd_util.go

```diff
@@ -142,8 +151,9 @@ func (util *RBDUtil) rbdLock(b rbdMounter, lock bool) error {
 		// for fencing, check if lock already held for this host
 		// this edge case happens if host crashes in the middle of acquiring lock and mounting rbd
 		// for defencing, get the locker name, something like "client.1234"
-		cmd, err = b.plugin.execCommand("rbd",
```

Comment:
replace it with `b.exec.Run` (like those in `DeleteImage` and `rbdStatus`) for quick diff